coderedcms/models/snippet_models.py

```diff
@@ -343,6 +343,14 @@ def __str__(self):
         return self.name
 
 
+@register_snippet
+class Accordion(ReusableContent):
```

Comment:
This should work exactly like the carousel... e.g. `Accordion` which contains many orderables of `AccordionItems` (for example).

I would also limit the accordion content to HTML_CONTENT, rather than the full nest of hero units, columns etc. It becomes unwieldy to edit/render.